src/main/java/org/opensearch/neuralsearch/highlight/SemanticHighlighter.java

```diff
@@ -60,8 +60,18 @@ public HighlightField highlight(FieldHighlightContext fieldContext) {
             return null;
         }
 
+        // The pre- and post- tags are provided by the user or defaulted to <em> and </em>
+        String[] preTags = fieldContext.field.fieldOptions().preTags();
+        String[] postTags = fieldContext.field.fieldOptions().postTags();
+
         // Get highlighted text - allow any exceptions from this call to propagate
-        String highlightedResponse = semanticHighlighterEngine.getHighlightedSentences(modelId, originalQueryText, fieldText);
+        String highlightedResponse = semanticHighlighterEngine.getHighlightedSentences(
+            modelId,
+            originalQueryText,
+            fieldText,
+            preTags[0],
```

Comment:
I think tag can be a list of string and we need to append them all instead of just using the first string in the list?

Reply:
Can you declare this preTag[0] to a variable at line 67 and then pass that variable here. Also provide an comment about why you are using first element.